### PR TITLE
Add materialize css

### PR DESCRIPTION
**Pull request by ahmedqarmout2**
This will include the library files needed for the new style to work. I also applied a sample of the new design on the login page.
canva-photo-editor 1

### CODE REVIEW

File: views/login.pug [hunk 1]
```diff
@@ -2,6 +2,8 @@ extends layout
 
 block styles
   link(rel='stylesheet' href='/css/login.css')
+  link(rel='stylesheet' href='/lib/materialize/css/materialize.min.css')
+  link(rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons")
```
Comment (Mohammed-JS-Khan) [inline views/login.pug]: Single quotes here for 'stylesheet' please.

Comment (Mohammed-JS-Khan) [reply views/login.pug]: And for the href, all on line 6 though.

**Review by Mohammed-JS-Khan — CHANGES_REQUESTED** [review]
Look pretty nice. Good find Ahmed. Just minor changes.

**Comment by Mohammed-JS-Khan** [comment]
See also #35, has redundant changes.

**Review by mehtadarshan1 — APPROVED** [review]
Looks good to me

**Review by Mohamed-A-Mohamed — APPROVED** [review]
New style looks great

**Review by Mohammed-JS-Khan — APPROVED** [review]

**Pull request merged**